ambry-account/src/test/java/com/github/ambry/account/HelixAccountServiceTest.java

```diff
@@ -202,6 +203,51 @@ public void testCreateAccount() {
     assertAccountsInAccountService(idToRefAccountMap.values(), NUM_REF_ACCOUNT, accountService);
   }
 
+  /**
+   * Tests generates a collection of reference {@link Account}s and {@link Container}s that can be referred
+   * from {@link #idToRefAccountMap} and {@link #idToRefContainerMap}. And it also generate a collection of
+   * the {@link Container}s through {@link HelixAccountService} with selected {@link ContainerStatus}
+   */
+  @Test
+  public void testGetContainerByStatus() throws Exception {
+    // a set that records the account ids that have already been taken.
+    Set accountIdSet = new HashSet<>();
+    // generate a single reference account and container that can be referenced by refAccount and refContainer respectively.
+    refAccountId = Utils.getRandomShort(random);
+    accountIdSet.add(refAccountId);
+    generateRefAccounts(idToRefAccountMap, idToRefContainerMap, accountIdSet, 5,
+        2);
+
+    accountService = mockHelixAccountServiceFactory.getAccountService();
+    accountService.updateAccounts(idToRefAccountMap.values());
+    assertAccountsInAccountService(idToRefAccountMap.values(), 5, accountService);
+
+    List<Account> accountsToUpdate = new ArrayList<>();
+    int cnt = 0;
+    for (Account account : accountService.getAllAccounts()) {
+      AccountBuilder accountBuilder = new AccountBuilder(account);
+      for (Container container : account.getAllContainers()) {
+        if (cnt%2 == 0) {
```

Comment:
Use `option + command + l` to format code.

Reply:
Thanks. Updated.